src/Shared/ProductCard/ProductCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Badge from '../Badge/Badge';
+import ratingIcon from '../../Assets/Icons/rating-icon.png' 
+import './ProductCard.css'
+
+const ProductCard = (props) => {
+    console.log("props:", props);
+    return(
+        <>
+            <div className="card-container card-shadow">
+                <img src={props?.product?.img ?? '-'}
```

Comment:
you can destructure props and use them, instead of props. everywhere

Reply:
ok